app.py

```diff
@@ -1,13 +1,127 @@
-from flask import Flask
-import requests
+from flask import Flask, request
+import requests, json, pymongo
+from pymongo import MongoClient
 
+def connect():
+    connection = MongoClient("ds031257.mlab.com",31257)
+    handle = connection["general_info_database"]
+    handle.authenticate("admin","admin1")
+    return handle
+
+handle=connect()
+db = handle.general_info_database
 app = Flask(__name__)
+URL = 'doc.json'
+db_users = handle.users_database
```

Comment:
This is :no_good:. The `users_database` is meant for authenticating access to mongo itself, not for authenticating users to the app. I would recommend looking into `WTForms` or something for the password authentication (or better yet, just use OAuth).